apps/modernization-ui/src/styles/_uswds.scss

```diff
@@ -102,3 +102,22 @@ table.usa-table {
 .usa-pagination__button:hover {
     border: 1px solid colors.$primary-dark !important;
 }
+
+// Trussworks tooltip
+.usa-tooltip {
+    // removing the button inserted by tooltip
+    .usa-button:has(svg) {
+        &.usa-tooltip__trigger {
+            background-color: white;
+            color: colors.$primary;
+            border: 2px solid colors.$primary;
+            padding: 0.4rem 0.75rem;
+        }
+    }
+    // styling tooltip body
+    .usa-tooltip__body {
+        font-family: "Public Sans";
+        font-size: 1rem;
+        padding: 0.75rem;
+    }
+}
```

Comment:
Issue: these are being applied globally. Anyone else in the app that uses the tooltip will have to fight with these style changes.

Reply:
this just strips out the most aggregious native styling by trussworks